### PR TITLE
MOL-66: Added GCS support to openbabel container

### PR DESCRIPTION
**Pull request by ada-phillips**
Added support for GCS storage of zipped results for openbabel
conversions
Moved archving process to happen on completion of task, rather than
on retrieval of results. This will save costs in GCS.
Added env variable control of GCS vs local storage, and .env files
for use with docker
Added .gitignore
Removed package-lock

### CODE REVIEW

**Review by kdm5498 — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -1,77 +1,95 @@
...
-    try{
-      archive.directory(fullPath, "", { name: jobId })
-      archive.finalize();
-    } catch(error) {
-      console.log(error)
+            if (useCloudStorage) {
+              output = storage.bucket(bucket).file(responsePath).createReadStream();
+            }
+            else {
+              output = fs.createReadStream(path.join(uploadsPath, jobId+'.zip'));
```
Comment (tydus101) [inline index.js:66]: Technically a security risk if we leave the ability to create files on the server - could take up all the memory. We should have a way to delete the files if we keep this.

File: index.js [hunk 2]
```diff
@@ -104,128 +122,178 @@ function openbabelFileConversion(req, res, outputName, options = [], inputFileTy
...
-  form.on('end', function() {
-    try {
-      //arguments for openbabel
-      let args = []
-
-      let execOptions = [] 
+    form.multiples = true;
+    form.parse(req);
+    form.on('field', function(name, value) {
+      fields[name] = value
```
Comment (tydus101) [inline index.js:162]: You don't define fields anywhere, probably should explicitly specify that with a let

File: index.js (hunk 1 above)
Comment (tydus101) [reply index.js:66]: Would still approve if you didn't fix this tho, since its out of our use case

**Review by tydus101 — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (ada-phillips) [reply index.js:66]: Yeah, given the use-case restrictions there, I'm gonna leave it be for now. Would add a `delete` endpoint if we had time, but it's not a huge deal.

File: index.js (hunk 2 above)
Comment (ada-phillips) [reply index.js:162]: It's defined with a `let` on line 121.

**Review by tydus101 — APPROVED** [review]
👌

**Pull request merged**